open-company/open-company-bot
# Update Slack digest design

[Trello](https://trello.com/c/oZzs0B0g)

This PR implements design changes, as well as a versioning scheme for Slack digest image assets in S3.

Update your local ENVs to use `DIGEST_BANNER_S3_BUCKET` rather than `AWS_S3_DIGEST_BANNER_BUCKET` and add an ENV for `DIGEST_FOOTER_S3_BUCKET` (bucket already created for you with the footer images in it).

Review this infrastructure change PR: https://github.com/belucid/open-company-infrastructure/pull/100

To test:

- Add a few new posts, make the very first one you add a Must See post
- Run a Slack digest for your user
- [x] Is the Must See post first in the digest?
- [x] Is the Must See marker at the end of the headline, not the beginning?
- [x] Do you have a randomize grey footer (run it multiple times)
- [x] Check your DIGEST_BANNER_S3_BUCKET with `-v1` at the end of it in S3. Was that the bucket used to create your banner? (file should be there with a timestamp of about now)
- Merge infra
- Merge this
- Deploy
- Make your St. Patrick's Day plans
